components/sendgrid/sendgrid.app.js

```diff
@@ -28,14 +60,30 @@ module.exports = {
       const authToken = this._authToken();
       const headers = {
         "Authorization": `Bearer ${authToken}`,
+        "Content-Type": "application/json",
         "User-Agent": "@PipedreamHQ/pipedream v0.1",
       };
       return {
         headers,
       };
     },
+    async _makeRequest(opts = {}) {
```

Comment:
Sendgrid has a well-maintained official API client. Can we use [sendgrid-nodejs](https://github.com/sendgrid/sendgrid-nodejs) instead of Axios?

Reply:
All Sendgrid action components were migrated to use sendgrid-nodejs instead of Axios.
Sendgrid sources, which aren't related to my assignemnt, weren't changed.